### PR TITLE
Feature/button

### PR DESCRIPTION
**Pull request by anyathegreat**

### CODE REVIEW

File: src/components/BaseSpinner.vue [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+.v-spinner-size-inline {
+  width: 1.2em;
+  height: 1.2em;
+}
+
+.v-spinner-color-dark {
+  --spinnerColor: #262626;
+  // & .path {
+  //   stroke: #262626;
+  // }
```
Comment (Ytar99) [inline src/components/BaseSpinner.vue]: Delete comments

File: src/components/BaseSpinner.vue [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+}
+.v-spinner-color-light {
+  --spinnerColor: #d9d9d9;
+  // & .path {
+  //   stroke: #d9d9d9;
+  // }
```
Comment (Ytar99) [inline src/components/BaseSpinner.vue]: Delete comments

**Review by Ytar99 — CHANGES_REQUESTED** [review]
Please, check my comments

File: src/components/BaseSpinner.vue (hunk 2 above)
Comment (anyathegreat) [reply src/components/BaseSpinner.vue]: Fixed

File: src/components/BaseSpinner.vue (hunk 1 above)
Comment (anyathegreat) [reply src/components/BaseSpinner.vue]: Fixed

**Review by Ytar99 — APPROVED** [review]

**Pull request merged**